introduction.md

```diff
@@ -1,4 +1,4 @@
 khanh vu
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?